chromium_src/storage/browser/blob/blob_url_store_impl.cc

```diff
@@ -120,7 +121,8 @@ void BlobURLStoreImpl::ResolveForNavigation(
 }
 
 GURL BlobURLStoreImpl::GetPartitionedOrOriginalUrl(const GURL& url) const {
-  if (!CanUseOriginForPartitioning(origin_)) {
+  url::Origin storage_key_origin = storage_key_.origin();
```

Comment:
`const url::Origin& storage_key_origin = storage_key_.origin();`

Reply:
Fixed in 4b99374e34